### ISSUE TITLE
RX 550 (Lexa/Polaris12 NOT Baffin 550) not flashable with atiflash

### ISSUE THREAD

**Issue by MoneroCrusher** [minershive/hiveos-linux#63]
Hi,

I can flash every card except my Lexa Pro cards (polaris12) (it wont even detect them "No adapter found"). The Windows version can detect and flash them but I'd like to do this on Linux with your 10+ card flasher. Any way you can implement this?
Thank you.

Kind Regards
MoneroCrusher

**Comment by minershive** [comment]
nope. atiflasher tool is a thirdparty tool which is unsupported by amd...

**Comment by MoneroCrusher** [comment]
Huh? You have atiflash in your distro.
Just like you implemented support for flashing 10+ cards, can you implement support for polaris12/gfx804 cards?

**Comment by minershive** [comment]
no, it's a completely different thing... sorry

**Comment by MoneroCrusher** [comment]
I had a closed sources program once that was not supporting polaris12/gfx804 and I did a hex search of it and it said ellesmere..baffin..etc
I simply replaced ellesmere with gfx804 and filled up with 0s and bam, it was them supporting my RX 550 Lexa pro. Unfortunately was not able to do this on atiflash but I believe conceptually it should be the same.
It looks in driver files for GPU name and them says yes or no and I believe gfx 804 is missing in the current atiflash.
I'm not a developer though and I'm optimistic that if you looked into it maybe something could be done..
There are lots of Monero folks with rx 550s
